src/libs/actions/Report.ts

```diff
@@ -2520,6 +2522,47 @@ function clearNewRoomFormError() {
     });
 }
 
+function resolveMentionWhisper(reportId: string, reportActionID: string, resolution: ValueOf<typeof CONST.REPORT.RESOLUTIONS>) {
```

Comment:
Can we call this `resolveActionableMentionWhisper`?

Let's call them `actionableMentionWhisper` instead of just `mentionWhisper` everywhere you're using this.